apache/metron
# METRON-1457: Move ASF links to main page in the Metron website

## Contributor Comments
Apache Whimsy checks for the site requisites in the main index.html and not inside sub-levels. This PR moves the content of the ASF and links to the main page.

Here is a screenshot of how the new metron page will look like:
https://imgur.com/3Y8ZLWL

**Testing Steps**
Follow the below steps to test the changes

* From `metron/site` execute
```
bundle install
bundle exec jekyll serve
```
* Navigate to http://localhost:4000 to review the site

Note that these steps are an excerpt from https://cwiki.apache.org/confluence/display/METRON/Website+PR+Merge. Please follow this page to have the pre-requisites set up.

## Pull Request Checklist

Thank you for submitting a contribution to Apache Metron.  
Please refer to our [Development Guidelines](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=61332235) for the complete guide to follow for contributions.  
Please refer also to our [Build Verification Guidelines](https://cwiki.apache.org/confluence/display/METRON/Verifying+Builds?show-miniview) for complete smoke testing guides.  


In order to streamline the review of the contribution we ask you follow these guidelines and ask you to double check the following:

### For all changes:


### For code changes:
  ```
  mvn -q clean integration-test install && dev-utilities/build-utils/verify_licenses.sh 
  ```


### For documentation related changes:

  ```
  cd site-book
  mvn site
  ```

#### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.
It is also recommended that [travis-ci](https://travis-ci.org) is set up for your personal repository such that your branches are built there before submitting a pull request.
